test/server/updater/testupdatechecker.cpp

```diff
@@ -34,7 +34,7 @@ static const std::string smallVersionJsonUpdateStr =
 class GetAppVersionJobTest final : public GetAppVersionJob {
     public:
         GetAppVersionJobTest(const Platform platform, const std::string &appID, const bool updateShouldBeAvailable) :
-            GetAppVersionJob(platform, appID), _updateShouldBeAvailable(updateShouldBeAvailable) {}
+            GetAppVersionJob(platform, appID, {}), _updateShouldBeAvailable(updateShouldBeAvailable) {}
```

Comment:
Would it be more natural to set the user id list with at least one element?

Reply:
I can change it, but it will not impact the test result. Moreover, no user ID is a valid case. For example, first installation where the user has not logged in yet

Reply:
It was not clear to me that an empty list isa valid use case; thanks for the clarification.

 Would it be possible then to use a version of the `GetAppVersionJob` constructor that does take the list as argument? 

The reason of my initial comment: the unit tests can also serve the purpose of code documentation. They ideally reflect actual use cases.

Reply:
Done in 95eeadc19f2f1300de7f391be3d7c5ddabd88413